### DISCUSSION TITLE
Managing/deleting PR comments

### DISCUSSION THREAD

**Discussion by brandonmcconnell** [community/community#15834] [Pull Requests]
It appears non-admin PR owners can delete admins’ comments on PRs & not the other way around. Admin status on a repo should always trump PR ownership in both cases—

* Admins should be able to delete any comments, especially their own
* Non-admins should not be able to delete admin comments, but should also be able to delete their own comments.

To be clear, this other user is the super-admin of our org and repo and has full write-access and even the access to remove my user from the repo. However, with all that access, I can delete his comments from my PR on the repo, yet **he cannot even delete his own comments**.

He doesn’t get that option in his drop-down. You can see the delete option I see from my account in this screenshot:

<img width="673" alt="Screen Shot 2022-04-30 at 07 57 56 " src="https://user-images.githubusercontent.com/5913254/166104952-ee76f4c3-12ab-4f15-bc03-b46f309ec026.png">

**Comment by tw4** [comment]
Absolutely I agree
